C/CUTEst/build_tarballs.jl

```diff
@@ -0,0 +1,131 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CUTEst"
+version = v"0.1.0"
+
+# Collection of sources required to build ThinASLBuilder
+sources = [
+    "https://github.com/ralna/ARCHDefs/archive/646f7c7b3848a547a70672243c1d75421268a8a8.zip" =>
+    "69d7afd52e2284a06e76f16a56b92e865490b149cfab578a0bb4a9058c9bf0f0",
```

Comment:
Oh, well, now the hash is different because the top-level directory is different:
```suggestion
    "8f340f515f84c5a5055968c148b1dcc77cfdc0e79d34dd8b9c8c3c11a027ee8e",
```

Since the top-level directory changed, you'll likely need to change some paths inside the script